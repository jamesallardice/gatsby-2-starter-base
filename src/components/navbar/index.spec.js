import React from 'react';
import { shallow } from 'enzyme';
import { CustomNavbar } from '.';

describe('[Component] Navbar', () => {
  it('renders as expected', () => {
    const data = {
      site: {
        config: {
          title: 'Example',
        },
      },
    };

    expect(shallow(<CustomNavbar data={data} />)).toMatchSnapshot();
  });
});
