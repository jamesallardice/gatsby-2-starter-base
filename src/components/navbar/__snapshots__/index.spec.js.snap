// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Component] Navbar renders as expected 1`] = `
<Navbar
  className="bg-primary"
  dark={true}
  expand="lg"
  tag="nav"
>
  <Container
    tag="div"
  >
    <NavbarBrand
      tag={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "render": [Function],
        }
      }
      to="/"
    >
      Example
    </NavbarBrand>
    <NavbarToggler
      onClick={[Function]}
      tag="button"
      type="button"
    />
    <Collapse
      appear={false}
      enter={true}
      exit={true}
      in={false}
      isOpen={false}
      mountOnEnter={false}
      navbar={true}
      onEnter={[Function]}
      onEntered={[Function]}
      onEntering={[Function]}
      onExit={[Function]}
      onExited={[Function]}
      onExiting={[Function]}
      tag="div"
      timeout={350}
      unmountOnExit={false}
    >
      <Nav
        className="ml-auto"
        navbar={true}
        tag="ul"
        vertical={false}
      >
        <NavItem
          tag="li"
        >
          <ForwardRef
            className="nav-link"
            to="/about"
          >
            About
          </ForwardRef>
        </NavItem>
      </Nav>
    </Collapse>
  </Container>
</Navbar>
`;
